src/vs/platform/extensionManagement/common/extensionManagementCLI.ts

```diff
@@ -22,6 +22,13 @@ const useId = localize('useId', "Make sure you use the full extension ID, includ
 type InstallVSIXInfo = { vsix: URI; installOptions: InstallOptions };
 type InstallGalleryExtensionInfo = { id: string; version?: string; installOptions: InstallOptions };
```

Comment:
With the suggestion in remoteExtensionsScanner to not to collect failed extensions here, we can undo all these changes in this file

Reply:
addressed in https://github.com/microsoft/vscode/pull/238815/commits/b356097a60e52c6eb95512ccfbb21cf9b62b1bc1